build-tools/installers/create-installers.targets

```diff
@@ -121,7 +121,6 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\cil-strip.pdb"   Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\class-parse.exe" Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\class-parse.pdb" Condition=" '$(PackageId)' != 'Microsoft.Android.Sdk' " />
-    <_MSBuildFiles Include="$(MSBuildSrcDir)\FSharp.Core.dll" />
```

Comment:
If we're removing this we may have a couple other areas that could be worth cleaning up:
https://github.com/xamarin/xamarin-android/blob/master/build-tools/create-vsix/symbols/ExternalWhiteList.csv#L9
and 
https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets#L273-L275